overrides-c9s.yaml

```diff
@@ -3,8 +3,14 @@
 # we need in the `packages` section. When not needed. Empty or comment out this
 # file (except this comment).
 
-#repos:
-#  - c9s-baseos-mirror
-#  - c9s-appstream-mirror
+repos:
+# - c9s-baseos-mirror
+  - c9s-appstream-mirror
 
-#packages:
+packages:
+  # https://issues.redhat.com/browse/RHEL-61612
+  - clevis-20-200.el9
+  - clevis-dracut-20-200.el9
+  - clevis-luks-20-200.el9
+  - clevis-systemd-20-200.el9
+  - clevis-udisks2-20-200.el9
```

Comment:
`clevis-udisks2` is not included in the image. Any reason to add it ?

Reply:
I do not think we should add it

Reply:
so is it not being pulled by any other packages? if so, I can remove it - but I'm not sure how we determine that.

Reply:
I checked on a local build and it's not pulled :) 
You can also look at the contents of a recent openshift RHCOS build : https://releases-rhcos-art.apps.ocp-virt.prod.psi.redhat.com/contents.html?stream=prod/streams/4.18-9.4&release=418.94.202410090804-0&arch=x86_64

Reply:
removed the package.